internal/js-ast-utils/isFunctionNode.test.ts

```diff
@@ -0,0 +1,60 @@
+import {test} from "rome";
+import {isFunctionNode} from "./isFunctionNode";
+import {template} from "./template";
+import {createBuilder} from "@internal/ast/utils";
+import {
+	AnyNode,
+	jsClassDeclaration,
+	jsVariableDeclarationStatement,
+} from "@internal/ast";
+
+test(
+	"returns true for all function node types",
+	async (t) => {
```

Comment:
```suggestion
	(t) => {
```

I don't see any async code (`await` keyword)